lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -5686,11 +5686,11 @@ func TestMakeRemapDotConfigRawRemapRangeDirective(t *testing.T) {
 
 	txtLines := strings.Split(txt, "\n")
 
-	if len(txtLines) != 3 { // 2 remaps plus header comment
+	if len(txtLines) != 4 { // 2 remaps plus header comment plus blank
 		t.Fatalf("expected 2 remaps from HTTP_AND_HTTPS DS, actual: '%v' count %v", txt, len(txtLines))
```

Comment:
> "expected <del>2</del><ins>4</ins> remaps from..."

Reply:
There are only 2 remaps, the other 2 lines are a comment and a blank